tests/page-objects/default/tasks/tasks.wdio.page.js

```diff
@@ -86,6 +88,12 @@ const openTaskById = async (id, taskType) => {
   await $(taskFormSelector).waitForDisplayed();
 };
 
+const getTaskHeaderOnMobile = async () => {
```

Comment:
Since we are moving the function to common, you can remove this one.

Reply:
Done